wgsl/index.bs

```diff
@@ -501,18 +520,49 @@ See [[#keyword-summary]] for the list of WGSL keywords.
 An <dfn>identifier</dfn> is a kind of [=token=] used as a name.
 See [[#declaration-and-scope]] and [[#directives]].
 
-The form of an identifier is defined via pattern-matching, except that
-an identifier:
-* must not have the same spelling as a [=keyword=] or as a [=reserved word=].
-* must not be `_` (a single underscore)
-* must not start with two underscores.
+The form of an identifier is based on the
+[=Unicode Standard Annex #31 for Unicode Version 14.0.0|Unicode Standard Annex #31=] for
+[[!UnicodeVersion14|Unicode Version 14.0.0]],
+with the following elaborations. Identifiers do not compare under canonical equivalence
+and they are not normalized otherwise,
+meaning that single accented code points are distinct from the same characters constructed
+by an accent and a letter, and Unicode code points that look similar are not
+treated as the same and they compare by their code points.
+
+Identifiers use the following profile described in terms of [=UAX31 Grammar=]:
+
+```
+<Identifier> := <Start> <Continue>* (<Medial> <Continue>+)*
+
+<Start> := XID_Start + U+005F
+<Continue> := <Start> + XID_Continue
+<Medial> :=
+```
+
+This means identifiers with non-ASCII characters like these are
+valid: `Δέλτα`, `réflexion`, `Кызыл`, `𐰓𐰏𐰇`, `朝焼け`, `سلام`, `검정`, `שָׁלוֹם`, `गुलाबी`, `փիրուզ`.
```

Comment:
For reviewers, these words are (according to Google translate....)

- "delta" in Greek
- "reflection" in French
- "Red" in Tatar
- ??
- "Sunrise" in Japanese
- "Hello" in Persian
- "black" in  Korean
- "Peace" in Hebrew
- "Rose" in Hindi
- "turquoise" in Armenian

Reply:
The question marks is "Good" or "Peaceful" in Turkic

Reply:
Not a comprehensive source but just as a pointer: https://en.wiktionary.org/wiki/𐰓𐰏𐰇